functions/c

```diff
@@ -32,7 +32,7 @@ function c() {
     cd $DIR
   fi
 
-  export CWD=pwd
+  export CWD=$(pwd)
```

Comment:
lol, yeah that makes way more sense...you can see how much I've been using this new feature  :)

could also use backticks...